packages/calcite-components/src/components/combobox-item-group/combobox-item-group.scss

```diff
@@ -49,3 +49,8 @@
   padding-inline: var(--calcite-combobox-item-spacing-unit-s);
   margin-inline-start: var(--calcite-combobox-item-indent-value);
 }
+
+.trailingPadding {
```

Comment:
can we change this to `padding--trailing` or `list-padding--trailing` as per BEM™ ?

Reply:
Directly quoting a Reddit comment I received a couple of days ago: "You won't get in trouble for not including the ™ btw" :)